From 8864d17169297580fb1135719dbf44a8ff4771d5 Mon Sep 17 00:00:00 2001
From: Tomohito Esaki <etom@igel.co.jp>
Date: Wed, 8 Aug 2018 14:20:46 +0900
Subject: [PATCH 2/2] drm: rcar-du: enable rcar-du render node

Create a render node tied to the DU device so that the allocated
buffers can be used for scanout, VSP composition, etc.

[dhobsong@igel.co.jp: Rebased to Linux 5.4]
Signed-off-by: Damian Hobson-Garcia <dhobsong@igel.co.jp>

[Fix for Resolve conflicts for BSP5.9 update]
Signed-off-by: Naoto Yamaguchi <naoto.yamaguchi@aisin.co.jp>
---
 drivers/gpu/drm/rcar-du/rcar_du_drv.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_drv.c b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
index 7389bbbad896..49272b9ec677 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_drv.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_drv.c
@@ -549,7 +549,8 @@ static const struct drm_ioctl_desc rcar_du_ioctls[] = {
 DEFINE_DRM_GEM_CMA_FOPS(rcar_du_fops);
 
 static struct drm_driver rcar_du_driver = {
-	.driver_features	= DRIVER_GEM | DRIVER_MODESET | DRIVER_ATOMIC,
+	.driver_features	= DRIVER_GEM | DRIVER_MODESET | DRIVER_ATOMIC
+				| DRIVER_RENDER,
 	.fops			= &rcar_du_fops,
 	.name			= "rcar-du",
 	.desc			= "Renesas R-Car Display Unit",
-- 
2.25.1

